### DISCUSSION TITLE
What happend to directus_roles.module_list?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12004] [Archive]
In early versions, we had a `module_list` in `directus_roles` collection [see for instance](https://github.com/directus/directus/discussions/10865#discussioncomment-1914146), but not anymore.

Why was it removed?

There are many use cases where we want to show modules accordingly to roles.

**Comment by azrikahar** [comment]
I believe this is a duplicate of #11217 and #9076. You can refer to this thread for more info on the reasoning: https://github.com/directus/directus/pull/8012#issuecomment-919094454. A slightly more recent discussion on this would be #10421 which you have commented as well 👍 Let's consolidate discussions so that they can be referred and tracked more easily.
